collectors/30/springboot.py

```diff
@@ -0,0 +1,60 @@
+!/usr/bin/env python
+import requests
+import sys
+import time
+
+from collectors.lib import utils
+from collectors.lib.optimizely_utils import format_tsd_key
+
+
+URL = 'http://127.0.0.1:8081/metrics'
+METRIC_PREFIX = 'springboot.'
+
+
+def request_json(url):
+    response = requests.get(url)
+    response.raise_for_status()
+    return response.json()
+
+
+def try_parse_value(value):
+    if isinstance(value, (int, long, float)):
+        return value
+
```

Comment:
in cases where the "metric" may be information, so things like which garbage collector is being used etc. Useful information but not something that is openTsdb-able